src/app/shared/dynamic-form/dynamic-form.component.scss

```diff
@@ -0,0 +1 @@
+// TODO : Add style here
```

Comment:
did you want to add styling as part of this PR? If yes, then add it, otherwise, don't just add an unnecessary `TODO` . You can add:
``` 
/* stylelint-disable */
```

Reply:
Added stylelint-disable.